common/src/main.c

```diff
@@ -36,6 +36,9 @@
 #include "models/models.h"
 #endif
 
+void donut(void);
```

Comment:
Can we create a `donut.h` header file so we can `#include "third_party/litex-donut/donut.h" instead of having this one-off prototype hear since none others are listed in this file like this?

Reply:
Sure; I was perhaps being too faithful to the original source (that's how they do it in the LiteX demo, which has a similar menu-driven structure).